theme/components/Tag/Tag.ts

```diff
@@ -15,6 +17,22 @@ const variants = {
   subtle: definePartsStyle((props) => ({
     container: Badge.variants?.subtle(props),
   })),
+  select: definePartsStyle((props) => ({
+    container: {
+      bg: mode('gray.100', 'gray.800')(props),
+      color: mode('gray.500', 'whiteAlpha.800')(props),
+      _hover: {
+        color: 'blue.400',
+        opacity: 0.76,
+      },
+    },
+  })),
+  selectActive: definePartsStyle((props) => ({
+    container: {
+      bg: mode('blue.500', 'blue.900')(props),
+      color: 'whiteAlpha.800',
+    },
+  })),
```

Comment:
I suppose `selectActive` should be a state for `select` variant, and not a separate variant.

```suggestion
  select: definePartsStyle((props) => ({
    container: {
      bg: mode('gray.100', 'gray.800')(props),
      color: mode('gray.500', 'whiteAlpha.800')(props),
      _hover: {
        color: 'blue.400',
        opacity: 0.76,
      },
      _active: {/* some styles */}
    },
  })),
```